google/resource_bigquery_dataset_test.go

```diff
@@ -87,6 +87,82 @@ func TestAccBigQueryDataset_access(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryDataset_regionalLocation(t *testing.T) {
+	t.Parallel()
+
+	datasetID1 := fmt.Sprintf("tf_test_%s", acctest.RandString(10))
+	datasetID2 := fmt.Sprintf("tf_test_%s", acctest.RandString(10))
+	datasetID3 := fmt.Sprintf("tf_test_%s", acctest.RandString(10))
+	datasetID4 := fmt.Sprintf("tf_test_%s", acctest.RandString(10))
+	datasetID5 := fmt.Sprintf("tf_test_%s", acctest.RandString(10))
+	datasetID6 := fmt.Sprintf("tf_test_%s", acctest.RandString(10))
+	datasetID7 := fmt.Sprintf("tf_test_%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigQueryDatasetDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBigQueryRegionalDataset(datasetID1, "asia-east1"),
```

Comment:
FYI we don't tend to test all our resources in every region they could be created in (at some point we have to assume that the API works as documented). Is there a reason we should treat bigquery datasets differently, or was this added just to be thorough?

Reply:
Just being thorough and as a way to test the validation.